### DISCUSSION TITLE
Download from Plex

### DISCUSSION THREAD

**Discussion by kodisMama** [Taxel/PlexTraktSync#1859] [Q&A] [answered]
Any idea how to use the download command?

![tempFileForShare_20240313-041256](https://github.com/Taxel/PlexTraktSync/assets/53911859/5021f2fa-664d-47a1-a18a-0f9b821356b8)

**Comment by glensc** [comment, marked as answer]
1. remove unrelated info from the post, things like how you can't type are not relevant in the question. text output is preferred
2. it takes same kind of input that `inspect` command does (read project readme)
3. downloading with compose it's important to map download directory out of container. this is out of scope of this project, how to use docker volumes, there's plenty of info in the internet

**Comment by kodisMama** [comment]
1. True, I'm really bad in it. Got it.
2. / 3. Thx
